modules/ROOT/pages/feature-flagging.adoc

```diff
@@ -370,6 +380,276 @@ Suppressed errors are treated as underlying causes that can also be matched by O
+
+* 4.5.0
+
+* 4.4.0-202202
+
+*Enabled by Default Since*
+
+* 4.5.0
+
+*Issue ID*
+
+* MULE-19967
+<.^|`mule.disable.registryBootstrapOptionalEntries`
+|When enabled, `optional` attribute in entries in a `registry-bootstrap.properties` are ignored.
+
+*Available Since*
+
+* 4.5.0
+
+*Enabled by Default Since*
+
+* 4.5.0
+
+*Issue ID*
+
+* W-10736301
+<.^|`mule.disable.applyObjectProcessor`
+|When enabled, `org.mule.runtime.core.privileged.registry.ObjectProcessor` implementations aren't applied on objects registered into the `SimpleRegistry`.
+
+*Available Since*
+
+* 4.5.0
+
+*Enabled by Default Since*
+
+* 4.5.0
+
+*Issue ID*
+
+* MULE-11737
+<.^|`mule.deployment.validateAppModelWithRegionClassloader`
+|When enabled, the application model is validated with the region classloader. When disabled, it is validated with the application classloader.
+
+*Available Since*
+
+* 4.5.0
+
+*Enabled by Default Since*
+
+* 4.5.0
+
+*Issue ID*
+
+* W-10808757
+<.^|`mule.suppress.mule.exceptions`
+|When enabled, error suppression happens. This affects, for example, the Web Service Consumer connector and the Until Successful scope that always reports errors from their corresponding namespaces (MULE and WSC). Suppressed errors are treated as underlying causes.
+
+*Available Since*
+
+* 4.5.0
+* 4.4.0-202210
+
+*Enabled by Default Since*
+
+* 4.5.0
+* 4.4.0-202210
+* 4.3.0-202210
+
+*Issue ID*
+
+* W-11308645
+<.^|`mule.foreachRouterRejectsMapExpressions`
+|When enabled, the 'foreach' router generates an `IllegalArgumentException` if the collection expression evaluates to a `java.util.Map`.
+
+*Available Since*
+
+* 4.5.0
+
+*Enabled by Default Since*
+
+* 4.5.0
+
+*Issue ID*
+
+* W-12207110
+<.^|`mule.honour.insecure.tls.configuration`
+|When enabled, the insecure TLS configuration is honored even if there are fields of the TrustStore configured.
+
+*Available Since*
+
+* 4.5.0
+
+*Enabled by Default Since*
+
+* 4.5.0
+
+*Issue ID*
+
+* W-10822938
+<.^|`mule.transaction.sink.index`
+|When enabled, flux sinks are cached using index as part of the key. If a sink is already in use, a new sink is created.
+
+*Available Since*
+
+* 4.4.0
+
+*Enabled by Default Since*
+
+* 4.0.0
+
+* 4.1.0
+
+* 4.2.0
+
+* 4.3.0
+
+* 4.4.0
+
+* 4.5.0
+
+*Issue ID*
+
+* W-12128703
+<.^|`mule.put.trace.id.and.span.id.in.mdc`
+|When enabled, the trace ID and span ID are added to the MDC when available.
+
+*Available Since*
+
+* 4.5.0
+
+*Enabled by Default Since*
+
+* 4.5.0
+
+*Issue ID*
+
+* W-12979787
+<.^|`mule.enable.mule.specific.tracing.information`
+|When enabled, the `ancestor-mule-span-id` value is added to the trace state when a span is propagated.
+
+*Available Since*
+
+* 4.5.0
+
+*Enabled by Default Since*
+
+* Not enabled by default in any Mule version
+
+*Issue ID*
+
+* W-13215870
+<.^|`mule.enable.policy.context.parallel.scopes`
+|When enabled, a new (Source) Policy Context is created for the execution of parallel scopes: ParallelForeach, ScatterGather, and Async.
```

Comment:
Are "ParallelForeach", "ScatterGather", and "Async" parameters or words that should be written in backticks?

Reply:
They are components. We don't write those with backticks